analyzer/src/funTest/assets/projects/synthetic/godep/lockfile/Gopkg.toml

```diff
@@ -0,0 +1,3 @@
+[[constraint]]
```

Comment:
Just thinking out aloud: So this new synthetic test project uses only one direct dependency. I hope this will not make us miss bugs that only appear with multiple direct dependencies.

Reply:
We only parse the GoDep lock file which contains a flat list of dependencies, so adding more dependencies here would not improve the test.